src/utils.js

```diff
@@ -466,7 +466,7 @@ exports.insertPixel = function (url) {
     } catch (e) {
     }
   };
-  this.insertElement(img);
+  exports.insertElement(img);
```

Comment:
For the record.... most of the functions in this file actually do this. I fixed a couple in the video cache PR too.

Some are easy to fix, but some aren't. Not sure which category `insertElement` is in, or how deep down the rabbit hole you want this to go... or if we should just file a ticket and schedule the time to fix them all.